benefits.md

```diff
@@ -52,28 +79,3 @@ Formula: `(regular work day hours - hours done on a sick day) * 80% + hours done
 
 Example: 2 hours work on a sick day. `(7.5h - 2h) * 0.8 + 2h = 6.4h`
 
-
-## Conferences
-
-Conferences count as working days (7.5h per day). Transport to conferences counts too (up to 7.5h per day).
-
-Try to give notice at least 10-14 days before attending a conference.
-
-Any expense for the conference, such as transport, accommodation, ticket, food, etc. is covered by your [allowance](#productivity-allowance). In order to not run out, you get a bonus for each conference day:
-* $50/day for local conferences (in your home town, or close)
-* $100/day for international conferences (you need to fly there, you need a hotel, etc.)
-* $200/day for intercontinental conferences (long flights, extra jetlag days, etc. )
-
-
-## Online and Offline Courses
-
-We encourage you to take courses by paying for time spent on the course. A rough guideline is one course per quarter. The amount covered is based on how direct the benefit is from the course, based on the course syllabus.
-
-* 50% for somewhat work related courses which do not have immediate benefit
-* 100% for work related courses with immediate benefit
-
-After finishing the course, you should write a short review of the course, for others to use when deciding whether to take a course or not.
-
-The main limitation is that project work always has priority over course-related work.
-
-If you need to pay a fee to participate in the course, use your [allowance](#productivity-allowance).
```

Comment:
Did you intentionally move `Conferences ` and `Online and Offline Courses` higher?

Reply:
Yes, because now everything that concerns allowance payouts is batched together.